Sources/Purchasing/Purchases.swift

```diff
@@ -1632,194 +1637,49 @@ public extension Purchases {
 public extension Purchases {
 
     /**
-     * Configures an instance of the Purchases SDK with a specified API key.
+     * Configures an instance of the Purchases SDK with a specified ``Configuration``.
      *
      * The instance will be set as a singleton.
      * You should access the singleton instance using ``Purchases/shared``
      *
-     * - Note: Use this initializer if your app does not have an account system.
-     * ``Purchases`` will generate a unique identifier for the current device and persist it to `NSUserDefaults`.
-     * This also affects the behavior of ``Purchases/restorePurchases(completion:)``.
-     *
-     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
-     *
-     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
-     */
-    @objc(configureWithAPIKey:)
-    @discardableResult static func configure(withAPIKey apiKey: String) -> Purchases {
-        configure(withAPIKey: apiKey, appUserID: nil)
-    }
-
-    /**
-     * Configures an instance of the Purchases SDK with a specified API key and app user ID.
-     *
-     * The instance will be set as a singleton.
-     * You should access the singleton instance using ``Purchases/shared``
-     *
-     * - Note: Best practice is to use a salted hash of your unique app user ids.
-     *
-     * - Warning: Use this initializer if you have your own user identifiers that you manage.
+     * - Note: Use this initializer if you wish to adjust network request timeouts.
```

Comment:
I think this isn't relevant anymore now that it's the main configure method?

Reply:
Oh yeah, good point.
✅